src/minivhd_util.h

```diff
@@ -82,4 +83,30 @@ uint32_t mvhd_gen_footer_checksum(MVHDFooter* footer);
  * \param [in] vhdm MiniVHD data structure
  */
 uint32_t mvhd_gen_sparse_checksum(MVHDSparseHeader* header);
+
+/**
+ * \brief Get current position in file stream
+ * 
+ * This is a portable version of the POSIX ftello64(). * 
+ */
+int64_t mvhd_ftello64(FILE* stream);
+
+/**
+ * \brief Reposition the file stream's position
+ * 
+ * This is a portable version of the POSIX fseeko64(). * 
+ */
+int mvhd_fseeko64(FILE* stream, int64_t offset, int origin);
+
+/**
+ * \brief Calculate the CRC32 of a data buffer.
+ * 
+ * This function can be used for verifying data integrity.
+ * 
+ * \param [in] data The data buffer
+ * \param [in] n_bytes The size of the data buffer in bytes
+ * 
+ * \return The CRC32 of the data buffer
+ */
+uint32_t mvhd_crc32(const void* data, size_t n_bytes);
```

Comment:
What usages of this do you envision? I have no objection to it, just interested in the use-cases.

Reply:
I forgot I left this in here. I used it for testing; it's how I found the bugs I mentioned previously. It could make it easier to test things in the future, but it's not necessary to keep it.

Reply:
Good enough for me 😁